test/AutoInstallRules_test.rb

```diff
@@ -135,10 +135,11 @@
           "result"      => { "profile"=>"machine12.xml" }
         }]
       )
+      env = { "hostaddress" => subject.hostaddress, "mac" => subject.mac }
```

Comment:
Maybe makes sense to assign the variable it in a before each block instead of assigning it 3 times...